lib/fastly_nsq/launcher.rb

```diff
@@ -40,25 +43,28 @@ def stopping?
   private
 
   def heartbeat
-    FastlyNsq.logger.debug do
+    logger.debug do
       [
         'HEARTBEAT:',
-        'thread_status:', @manager.listeners.map(&:status).join(', '),
-        'listener_count:', @manager.listeners.count
+        'busy:', manager.pool.length,
+        'processed:', manager.pool.completed_task_count,
+        'max_threads:', manager.pool.max_length,
+        'max_queue_size:', manager.pool.largest_length,
+        'listeners:', manager.listeners.count
```

Comment:
This is great!
I wonder if there is interesting information on the `Priorities` that we can add in in the future.

Reply:
We definitely could